Description: fix lintian appstream-metadata-in-legacy-location
Author: ghost <i@ghosts.work>
Last-Update: 2018-10-10
--- a/ghostwriter.pro
+++ b/ghostwriter.pro
@@ -261,7 +261,7 @@
     desktop.path = $$DATADIR/applications/
 
     appdata.files = resources/linux/ghostwriter.appdata.xml
-    appdata.path = $$DATADIR/appdata/
+    appdata.path = $$DATADIR/metainfo/
 
     man.files = resources/linux/ghostwriter.1
     man.path = $$PREFIX/share/man/man1
